src/visualization/maps/ReconMap/generateSubsystemLayouts.m

```diff
@@ -17,9 +17,9 @@
     for i= 1:length(subsystems)
```

Comment:
This would need to be adapted to the current subSystems scheme.
There is currently an open PR ( #1010 ) which provides utility functions to work with subSystems (e.g. a utility to extract all present subSystems `getModelSubSystems()`), which could be used here.